ETC.tsv

```diff
@@ -19582,171 +19582,171 @@ ETC_20151224_019583
-ETC_20151224_019617
-ETC_20151224_019618
-ETC_20151224_019619
-ETC_20151224_019620
-ETC_20151224_019621
-ETC_20151224_019622
-ETC_20151224_019623
-ETC_20151224_019624
-ETC_20151224_019625
-ETC_20151224_019626
-ETC_20151224_019627
-ETC_20151224_019628
-ETC_20151224_019629
-ETC_20151224_019630
-ETC_20151224_019631
-ETC_20151224_019632
-ETC_20151224_019633
-ETC_20151224_019634
-ETC_20151224_019635
-ETC_20151224_019636
-ETC_20151224_019637
-ETC_20151224_019638
-ETC_20151224_019639
-ETC_20151224_019640
-ETC_20151224_019641
-ETC_20151224_019642
-ETC_20151224_019643
-ETC_20151224_019644
-ETC_20151224_019645
-ETC_20151224_019646
-ETC_20151224_019647
-ETC_20151224_019648
-ETC_20151224_019649
-ETC_20151224_019650
-ETC_20151224_019651
-ETC_20151224_019652
-ETC_20151224_019653
-ETC_20151224_019654
-ETC_20151224_019655
-ETC_20151224_019656
-ETC_20151224_019657
-ETC_20151224_019658
-ETC_20151224_019659
-ETC_20151224_019660
-ETC_20151224_019661
-ETC_20151224_019662
-ETC_20151224_019663
-ETC_20151224_019664
-ETC_20151224_019665
-ETC_20151224_019666
-ETC_20151224_019667
-ETC_20151224_019668
-ETC_20151224_019669
-ETC_20151224_019670
-ETC_20151224_019671
-ETC_20151224_019672
-ETC_20151224_019673
-ETC_20151224_019674
-ETC_20151224_019675
-ETC_20151224_019676
-ETC_20151224_019677
-ETC_20151224_019678
-ETC_20151224_019679
-ETC_20151224_019680
-ETC_20151224_019681
-ETC_20151224_019682
-ETC_20151224_019683
-ETC_20151224_019684
-ETC_20151224_019685
-ETC_20151224_019686
-ETC_20151224_019687
-ETC_20151224_019688
-ETC_20151224_019689
-ETC_20151224_019690
-ETC_20151224_019691
-ETC_20151224_019692
-ETC_20151224_019693
-ETC_20151224_019694
-ETC_20151224_019695
-ETC_20151224_019696
-ETC_20151224_019697
-ETC_20151224_019698
-ETC_20151224_019699
-ETC_20151224_019700
-ETC_20151224_019701
-ETC_20151224_019702
-ETC_20151224_019703
-ETC_20151224_019704
-ETC_20151224_019705
-ETC_20151224_019706
-ETC_20151224_019707
-ETC_20151224_019708
-ETC_20151224_019709
-ETC_20151224_019710
-ETC_20151224_019711
-ETC_20151224_019712
-ETC_20151224_019713
-ETC_20151224_019714
-ETC_20151224_019715
-ETC_20151224_019716
-ETC_20151224_019717
-ETC_20151224_019718
-ETC_20151224_019719
-ETC_20151224_019720
-ETC_20151224_019721
-ETC_20151224_019722
-ETC_20151224_019723
-ETC_20151224_019724
-ETC_20151224_019725
-ETC_20151224_019726
-ETC_20151224_019727
-ETC_20151224_019728
-ETC_20151224_019729
-ETC_20151224_019730
-ETC_20151224_019731
-ETC_20151224_019732
-ETC_20151224_019733
-ETC_20151224_019734
-ETC_20151224_019735
-ETC_20151224_019736
-ETC_20151224_019737
-ETC_20151224_019738
-ETC_20151224_019739
-ETC_20151224_019740
-ETC_20151224_019741
-ETC_20151224_019742
-ETC_20151224_019743
-ETC_20151224_019744
-ETC_20151224_019745
-ETC_20151224_019746
-ETC_20151224_019747
-ETC_20151224_019748
-ETC_20151224_019749
-ETC_20151224_019750
-ETC_20151224_019751
+ETC_20151224_019587	Mining...
+ETC_20151224_019588	You cannot mine here.
+ETC_20151224_019589	Only guild members within guilds with the mining skill can mine.
```

Comment:
Only guild members with the mining skill can mine.

or

Only people with the mining skill who are apart of the guild can mine (literal)